tensorflow-tools/src/main/java/org/tensorflow/tools/Shape.java

```diff
@@ -148,6 +278,51 @@ public Shape prepend(long firstDimension) {
     return Shape.of(newDimensions);
   }
 
+  /** Returns a new shape, with a new last dimension added. In order for this call to succeed,
+   * {@link Shape#isUnknown()} must be {@code false}.
+   * @param lastDimension the dimension to append
+   * @return a new shape with this shape's dimensions followed by the given dimension
+   */
+  public Shape append(long lastDimension) {
+    long[] newDimensions = new long[dimensionSizes.length + 1];
+    newDimensions[newDimensions.length - 1] = lastDimension;
+    System.arraycopy(dimensionSizes, 0, newDimensions, 0, dimensionSizes.length);
+
+    return Shape.of(newDimensions);
+  }
+
+  /** Returns a new shape, with another shapes' dimensions prepended.
```

Comment:
`shape's`? same thing for `append(Shape)`

Reply:
I Really should have proof-read before committing...